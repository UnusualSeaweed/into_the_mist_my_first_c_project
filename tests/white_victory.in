place A10
place A19
place S1
place B18
place S2
place C17
place S3
place D16
place S4
place E15
