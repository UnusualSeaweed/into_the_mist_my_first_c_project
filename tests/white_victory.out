White wins!
A10A19S1B18S2C17S3D16S4E15
Thank you for playing!
